prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -17,6 +19,10 @@ asap_accepted_issuers = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_ISSU
 asap_accepted_audiences = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_AUDIENCES) }}" }
 {{ end }}
 
+{{ if $ENABLE_XMPP_WEBSOCKET }}
+cross_domain_websocket = { "{{ .Env.PUBLIC_URL }}" };
```

Comment:
I think we set this to:
```
cross_domain_websocket =  true;
consider_websocket_secure = true;
```
But if it works without the `consider_websocket_secure` option then I guess it's fine.

Reply:
Maybe you won't need the `PUBLIC_URL` with these  options?

<img width="783" alt="Screenshot 2020-04-23 18 22 37" src="https://user-images.githubusercontent.com/2965063/80158939-72bc9300-858f-11ea-8369-e097a1ca0ea0.png">

Reply:
> Maybe you won't need the PUBLIC_URL with these options?

See above: The web component needs a `PUBLIC_URL`.

```
cross_domain_websocket = true;
```

I learned that allow any origin in CORS policies is an anti pattern.